### DISCUSSION TITLE
Archive/hide habit option

### DISCUSSION THREAD

**Discussion by justeepin** [iSoron/uhabits#765] [Feature Requests] [answered]
How about adding an option to archive/hide a habit in addition to delete? That way if there's a habit that you need to temporarily suspend or are finished but want to keep the data you have a way to do so.

**Comment by iSoron** [comment, marked as answer]
Thank you for the feature request, @justeepin, but Loop already has this feature!
Archiving a habit currently hides it from the main screen and disables its reminders.

**To archive a habit:**
1. In the main screen, long-press the name of a habit to select it
2. Click the "..." icon in the menu
3. Click "Archive"

**To unarchive a habit:**
1. Open the app
2. Tap the filter icon (the one with three lines) then uncheck "Hide archived"
3. Long-press the archived habit
4. Click the "..." icon in the menu
5. Click "Unarchive"

**Reply by NomarCub** [reply]
Could you set this as the answer?

**Comment by justeepin** [comment]
Thank you.  I did look in the FAQ and didn't see anything about archiving.
